demos/browser/app/messagingSession/messagingSession.html

```diff
@@ -28,6 +28,11 @@ <h1 class="h3 mb-3 font-weight-normal">Messaging Session</h1>
             Connect
           </button>
         </div>
+        <div class="row mt-3">
```

Comment:
We should not do another button for this? Can you look at the meeting demo and add a check box to turn on pre fetch when connect?

Reply:
Thanks addressed your comment.